stdlib/importlib/metadata/__init__.pyi

```diff
@@ -35,6 +35,16 @@ if sys.version_info >= (3, 8):
             dist: ClassVar[Distribution | None]
             def matches(self, **params: Any) -> bool: ...  # undocumented
 
+    if sys.version_info >= (3, 10):
+        class SelectableGroups(dict):  # use as list is deprecated since 3.10
+            @classmethod
+            def load(cls, eps: SelectableGroups) -> SelectableGroups: ...
```

Comment:
I'm not sure if `eps` should be `SelectableGroups` / `EntryPoints` / both.

Reply:
Maybe `SelectableGroups` should extend `EntryPoints`?

Reply:
> Maybe `SelectableGroups` should extend `EntryPoints`?

Never mind, that doesn't make sense given the current implementation in CPython.